api/models/cluster.go

```diff
@@ -24,3 +24,18 @@ type GetDataResp struct {
 type MetricsDeleteResp struct {
 	DeletedDefs int `json:"deletedDefs"`
 }
+
+//go:generate msgp
+type IndexTagsResp struct {
+	Tags []string `json:"tags"`
+}
+
+//go:generate msgp
+type IndexTagDetailsResp struct {
+	Values map[string]uint64 `json:"values"`
+}
+
+//go:generate msgp
+type IndexTagFindSeriesResp struct {
+	Series []string `json:"series"`
```

Comment:
the docs for FindByTag talk about "metric names", when we call it we do "metrics, err := s.MetricIndex.FindByTag"
and here we talk about series. can we call this "metrics" here too?

Reply:
done :+1: